connector/oidc/oidc.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 
 	Scopes []string `json:"scopes"` // defaults to "profile" and "email"
 
+	HostedDomain string `json:"hostedDomain"`
```

Comment:
Please add a comment describing what this field is and highlighting that it is optional